test/path-parsing.spec.js

```diff
@@ -47,8 +47,7 @@ describe('gulp-rename path parsing', function () {
       dirnameHelper('test/fixt[a-z]res/*.min.txt');
     });
 
-    /* SKIP: glob2base does not handle brace expansion as expected. See wearefractal/glob2base#1 */
-    context.skip('when src pattern matches a directory with {...,...}', function () {
+    context('when src pattern matches a directory with {...,...}', function () {
```

Comment:
The one in between these two still fails. Is that test good? From my understanding, it should match a folder with a numerical postfix, which doesn't exist anyways.